schutzbot/ci_details.sh

```diff
@@ -37,6 +37,13 @@ echo "List of installed packages:"
 rpm -qa | sort
 echo "------------------------------------------------------------------------------"
 
+if [ $(rpm -qa | grep -c python36) == 0 ]; then
```

Comment:
I think you can simplify this to:
```suggestion
if rpm --quiet -q python36; then
```

That also gets rid of the shellcheck warning.